test/jdk/com/sun/java/swing/plaf/gtk/TestMenuMnemonicOnAltPress.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8326458
+ * @key headful
+ * @requires (os.family == "linux")
+ * @summary Verifies if menu mnemonic toggle on Alt press in GTK LAF
+ * @run main TestMenuMnemonicOnAltPress
+ */
+
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+import java.util.concurrent.atomic.AtomicInteger;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.plaf.synth.SynthLookAndFeel;
+
+public class TestMenuMnemonicOnAltPress {
```

Comment:
This test seems to repeat the [`JMenuBar/TestMenuMnemonic.java`](https://github.com/openjdk/jdk/blob/master/test/jdk/javax/swing/JMenuBar/TestMenuMnemonic.java) test that you created for Windows L&F when you worked on [JDK-8326458](https://bugs.openjdk.org/browse/JDK-8326458).

Reply:
This test is almost similar to the [JMenuBar/TestMenuMnemonic.java](https://github.com/openjdk/jdk/blob/master/test/jdk/javax/swing/JMenuBar/TestMenuMnemonic.java) but is bit different for linux. 
Here we are testing for Alt key press scenario whereas windows test was checking for F10 key press. And secondly windows test checked for the menu selection path also which is not the case in linux.

Initially I thought of extending the same test for linux fix but then decided not to do it for not making it too much complex.